cmd/sheltertech-go/main.go

```diff
@@ -50,12 +53,24 @@ func main() {
 	dbName := viper.GetString("DB_NAME")
 	dbPass := viper.GetString("DB_PASS")
 	serveDocs := viper.GetBool("SERVE_DOCS")
+	auth0Domain := viper.GetString("AUTH0_DOMAIN")
```

Comment:
It looks like this isn't secret, so I'm going to push up a commit to add this to our docker-compose.yml file, so that local development should work seamlessly. We'll still need to add this to our deployment configuration files, though.

Reply:
Added in 4bb98b7c81ea8a235a7d6cea2120ab0b51321b88.